src/main/resources/assets/securitycraft/lang/ko_kr.json

```diff
@@ -618,41 +728,70 @@
+	"messages.securitycraft:universalKeyChanger.briefcase.passcodeReset": "이 서류 가방의 비밀번호가 재설정되었습니다.",
+	"messages.securitycraft:universalKeyChanger.briefcase.noPasscode": "이 서류 가방에는 비밀번호가 없으므로 재설정할 수 없습니다.",
+	"messages.securitycraft:universalKeyChanger.briefcase.notOwned": "이 서류 가방은 귀하의 서류가방이 아니므로 비밀번호를 변경할 수 없습니다.",
+	"messages.securitycraft:passwordProtected.notSetUp": "이 블록에는 아직 비밀번호가 없습니다. 당신은 그것과 상호작용을 할 수 없습니다.",
+	"messages.securitycraft:sentry.needsBlockBelow": "당신은 센트리 건를 블록 위에 올려놓아야 합니다!",
+	"messages.securitycraft:sentry.needsBlocksAround": "센트리 건가 모든 네 면에 블록으로 둘러싸여 있는지 확인합니다!",
+	"messages.securitycraft:sentry.mode0": "현재 모드: 공격 ",
+	"messages.securitycraft:sentry.mode1": "현재 모드: 위장 ",
+	"messages.securitycraft:sentry.mode2": "현재 모드: 유휴 ",
+	"messages.securitycraft:sentry.descriptionMode0": "(항상 활성화되며 적대적인 몹과 플레이어를 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode1": "(적대적인 몹이나 플레이어가 가까이 오면 활성화되며 적대적인 몹과 플레이어를 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode2": "(항상 활성화되며 적대적인 몹을 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode3": "(적대적인 몹이 가까이 왔을 때 활성화되며 적대적인 몹을 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode4": "(항상 활성화되며 플레이어를 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode5": "(플레이어가 가까이 오면 활성화되며 플레이어를 공격합니다.)",
+	"messages.securitycraft:sentry.descriptionMode6": "(활성화되지 않고, 공격하지 않습니다.)",
+	"messages.securitycraft:blockpocket.multipleManagers": "구조물에 두 개 이상의 블록 포켓 매니저가 있습니다.",
+	"messages.securitycraft:blockpocket.alreadyAssembled": "블록 포켓이 이미 조립되었습니다!",
+	"messages.securitycraft:blockpocket.invalidBlock.rotation": "기둥 %1$s 블록 포켓 매니저가 잘못 회전됩니다!",
+	"messages.securitycraft:blockpocket.invalidBlock": "잘못된 블록 \"%2$s\" %1$s 블록 포켓 매니저! (예상: \"%3$s\")",
+	"messages.securitycraft:blockpocket.blockInWay": "\"%2$s\"이(가) 공간 %1$s 블록 포켓 매니저를 차단하고 있습니다!",
 	"messages.securitycraft:blockpocket.unowned": "%1$s 위치에 블록 \"%2$s\"을(를) 소유하고 있지 않습니다!",
+	"messages.securitycraft:blockpocket.position.1": "%s",
+	"messages.securitycraft:blockpocket.position.2": "%1$s, %2$s",
+	"messages.securitycraft:blockpocket.position.3": "%1$s, %2$s, %3$s",
+	"messages.securitycraft:blockpocket.position.blocksLeft": "%s 블록의 왼쪽",
+	"messages.securitycraft:blockpocket.position.blocksRight": "%s 블록의 오른쪽",
+	"messages.securitycraft:blockpocket.position.blocksBehind": "%s 블록의 뒷쪽",
+	"messages.securitycraft:blockpocket.position.blocksAbove": "%s 블록의 앞쪽",
+	"messages.securitycraft:blockpocket.notEnoughItems": "이 크기의 블록 포켓을 조립할 수 있는 아이템이 부족합니다!",
 	"messages.securitycraft:blockpocket.activated": "블록 포켓이 활성화되었습니다!",
 	"messages.securitycraft:blockpocket.deactivated": "블록 포켓이 비활성화되었습니다!",
+	"messages.securitycraft:blockpocket.assembled": "블록 포켓이 조립되었습니다!",
+	"messages.securitycraft:blockpocket.assemblyFailed": "블록 포켓 조립이 %1$s 블록 포켓 매니저로 인해 실패했습니다. (잘못된 블록: %2$s)",
+	"messages.securitycraft:sc_help": "%s: %s + %s",
+	"messages.securitycraft:formattedCoordinates": "X: %1$s Y: %2$s Z: %3$s",

```

Comment:
These are unchanged to en_us, please remove.

Reply:
You don't have to remove it.